### ISSUE TITLE
generate data

### ISSUE THREAD

**Issue by zjxyz** [Tong-ZHAO/Pixel2Mesh-Pytorch#12]
hi,
I wonder if you can provide code to generate training data.
when  I run obj_path='ShapeNetCore.v1/02691156/1a04e3eab45ca15dd86060f189eb133/model.obj'
mesh_list=trimesh.load_mesh(obj_path),
,it is ok.
But I run
obj_path='ShapeNetCore.v1/02691156/10155655850468db78d106ce0a280f87/model.obj
mesh_list=trimesh.load_mesh(obj_path),
ValueError:all the input array dimensions except for the concatenation axis must match exactly.
